color_in_sequence.py

```diff
@@ -0,0 +1,130 @@
+
+# library for waiting for network operations to finish
+import asyncio 
+
+# Just for knowing what types you can enter in your IDE
+from typing import List, Tuple 
+
+# Custom util I made for saving bulbs.
+# This function gets you the lights directly via a name
+from utils.bulb_groups import get_wiz_light_from_group
+
+# PilotBuilder makes colors and settings
+# Wizlight is how you make calls to a light
+from pywizlight import PilotBuilder, wizlight
+
+# You know this one
+from time import sleep
+
+# continually iterates through an array
+from itertools import cycle, islice
+
+
+assortment_of_color_rgbs = [
+	(0,0,255),
+	(0,255,0),
+	(0,255,255),
+	(255,0,0),
+	(255,0,255),
+	(255,255,0),
+	(255,255,255),
+]
+
+metro_palette = [
+	(0,174,219),
+	(162,0,255),
+	(244,120,53),
+	(212,18,67),
+	(142,193,39)
+]
+
+brighter_palette = [
+	(212,18,67),
+	(0,19,222),
+	(5,213,250),
+	(243,119,53),
+	(50,205,50)
+]
+
+rgb_palette = [
+	(0, 0, 255),
+	(0, 255, 0),
+	(255, 0, 0)
+]
+
+async def turn_all_off(bulbs: List[wizlight]):
+	for bulb in bulbs:
+		await bulb.turn_off()
+
+async def turn_all_to_color(bulbs: List[wizlight], rgb_color_tuple: Tuple[int, int, int] ):
+	# make the color type that wizlights use
+	color = PilotBuilder(rgb=rgb_color_tuple)
+	for bulb in bulbs:
+		await bulb.turn_on(color)
+
+#cycle through colors and change lights sequentially
+async def color_sequence(bulbs: List[wizlight]):
+	# make the color type that wizlights use
+	for rgb_color_tuple in cycle(brighter_palette):
+		for bulb in bulbs:
+			color = PilotBuilder(rgb=rgb_color_tuple)
+			await bulb.turn_on(color)
+			sleep(0.5)
+
+#cycle through colors and change all three lights at the same time
```

Comment:
Nitpick:
All of these comments could be doc strings in the function and then they will display in the IDE.